<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportManager</class>
 <widget class="QWidget" name="ReportManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="dailySalesButton">
     <property name="text">
      <string>刷新每日销售额报表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="dailySalesTableWidget">
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>销售总额</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="menuItemSalesButton">
     <property name="text">
      <string>刷新菜品销量报表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="menuItemSalesTableWidget">
     <column>
      <property name="text">
       <string>菜品名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>销量总数</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
